ui/app/components/app/gas-customization/gas-price-chart/tests/gas-price-chart.component.test.js

```diff
@@ -0,0 +1,258 @@
+import assert from 'assert'
```

Comment:
It looks like there was a problem during rebase - this file was deleted in another PR, and shouldn't be re-introduced here